index.html

```diff
@@ -19,17 +19,20 @@ <h2 id="winnerMessage">Choose your option to see who wins</h2>
 
         <div class="choices">
             <div class="content" data-playerChoices="Rock">
-                <button class="selection"><img src="./images/rock.png" alt="Rock">
+                <button class="selection">
+                        <img src="./images/rock.png" alt="Rock">
                     <div class ="text">Rock</div>
                 </button>
             </div>
             <div class="content" data-playerChoices="Paper">
-                <button class="selection"><img src="./images/paper.png" alt="Paper">
+                <button class="selection">
+                        <img src="./images/paper.png" alt="Paper">
                     <div class ="text">Paper</div>
                 </button>  
             </div>
             <div class="content" data-playerChoices="Scissors">
-                <button class="selection"><img src="./images/scissors.png" alt="Scissors">
+                <button class="selection">
+                        <img src="./images/scissors.png" alt="Scissors">
```

Comment:
All these img tags inside buttons should be veritically lined up with the div below it